doc/quickstart/udig_quickstart.rst

```diff
@@ -32,9 +33,10 @@ This Quick Start describes how to:
   * apply colour to map features using styling
 
 .. contents:: Contents
+   :local:
   
 start uDig
-================================================================================
+==========
 
 .. Writing Tip:
   Describe steps to start the application
```

Comment:
Now that we have a _template_quickstart, I think we can remove all the embedded writing tips from here but I will check before I do that.

Reply:
sounds good, this was really just an early example